mottak-arkiv-web/src/components/common/ProcessStatusButton.tsx

```diff
@@ -0,0 +1,67 @@
+import { Help } from '@material-ui/icons'
+import { Button } from '@material-ui/core'
+import styled from 'styled-components'
+import Tippy from '@tippyjs/react'
+
+import Status from './Status'
+
+/**
+ * The ProcessStatusButton is a generic component that supports a click (that typically make a server call)
+ * and show a status field.
+ */
+
+const Wrapper = styled.div``
+
+const Title = styled.h3`
+	display: flex;
+	align-items: center;
+	margin-bottom: 0.25rem;
+
+	span {
+		margin-right: 0.5rem;
+	}
+
+	svg {
+		cursor: help;
+	}
+`
+
+type Props = {
+	children?: React.ReactNode
+	topTitle: string
+	buttonTitle: string | React.ReactNode
+	helpText: string | React.ReactNode
+	statusLine: string | null
+	disabled: boolean
+	onClick: () => void
+}
+
+const ProcessStatusButton = ({
+	children,
+	topTitle,
+	buttonTitle,
+	helpText,
+	statusLine,
+	disabled,
+	onClick,
+}: Props): JSX.Element => {
+	return (
+		<Wrapper>
+			<Title>
+				<span>{topTitle}</span>
+				<Tippy content={helpText}>
+					<Help color="action" fontSize="small" />
+				</Tippy>
+			</Title>
+			<Status loading={statusLine == null} status={statusLine} />
+
+			<Button variant="contained" color="primary" disabled={disabled} onClick={onClick}>
+				{buttonTitle}
```

Comment:
```suggestion
				{children}
```
Erstatt `buttonTitle` med `children`

Reply:
Min tanke var at støtte for ekstra innhold under knappen var mer påkrevd enn HTML inni knappen.

Reply:
Kommer nok ikke til å bli nødvendig